README.md

```diff
@@ -11,15 +10,20 @@ Install cookiecutter.
 pip install cookiecutter
 ```
 
-Use it to generate a package, following prompts fill in the name and authorship
-of your new JupyterLab extension.
+Use it to generate a package, following the prompts to fill in the name and authorship of your new JupyterLab extension.
 
 ```
 cookiecutter https://github.com/jupyter/jupyterlab-extension-cookiecutter
 ```
 
-A very simple working extension, written in common JavaScript, is included in
-the ``lib/`` directory. Use this example to build your own extension.
+A very simple working extension, written in Javascript using CommonJS modules, is included in the ``lib/`` directory. Use this example to build your own extension.
```

Comment:
Let's flip line 19 and 21.

Add a heading:

 ### A simple example

The ``lib/`` directory includes a very simple example of a working extension, written in JavaScript using CommonJS modules. Use this example as a guide to build your own extension.